index.html

```diff
@@ -0,0 +1,178 @@
+        <header class="main-body">
+          <div class="tools">
+            <nav class="main-nav">
+              <div class="items">
+                <a class="logo" href="#head"><img src="./images/afri-tech-img.png" alt="logo"></a>
+                <div class="mobile" id="harmburger">
+
+                  <img onclick="clickToSwitch()" alt="harmburger" src="images/harmburger_img.png" />
+
+                </div>
+                <div class="desktop">
+                  <ul class="nav-icons">
+                    <li><a href="#program">Program</a></li>
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#join">Join</a></li>
+                    <li><a href="#sponsor">Sponsor</a></li>
+                    <li><a href="#news">News</a></li>
+                  </ul>
+                </div>
+              </div>
+
+              <!--Mobile blue menu-->
+            <div class="mobile-menu-container" id="switch-display">
+              <div class="mobile-items">
+                <div id="mobile-x-icon">
+                  <img onclick="clickToSwitch()" alt="close-icone-image" src="images/x-icon.png" />
+                </div>
+                <div class="mobile-menu">
+                  <ul class="mobile-list">
+                    <li><a onclick="clickToSwitch()" href="#program">Program</a></li>
+                    <li><a onclick="clickToSwitch()" href="./about.html">About</a></li>
+                    <li><a onclick="clickToSwitch()" href="#join">Join</a></li>
+                    <li><a onclick="clickToSwitch()" href="#sponsor">Sponsor</a></li>
+                    <li><a onclick="clickToSwitch()" href="#news">News</a></li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </nav>
+          </div>
+        </header>
+        <section class="head-part" id="head">
+          <h1 class="mobile">
+            About Africa<br/>Tech
+            Week
+          </h1>
+          <h2 class="head-h2 desktop">
+            About Africa <br/>Tech
+            Week
+          </h2>
+          <p class="head-part-text">
+            Africa Tech Week is a technology conference, an exhibition and a Technology Awards Ceremony. 
+            It focuses on educating South Africa and other African countries on the challenges and opportunities 
+            to be gained by their drafting appropriate plans and policies for the 4th Industrial revolution. 
+            Africa Tech Week will also recognize and reward companies on the continent that have demonstrated 
+            excellence in the area of innovative product development. The speed of current breakthroughs has no historical precedent.
+          </p>
+          <h4 class="head-h4">LET'S CONNECT</h4>
+          <h4 class="head-h4" ><strong>25-29 OCTOBER 2022</strong></h4>
+          <h4 class="head-h4">Virtual Event</h4>
+        </section>
+      </div>
+      <!-- Card section Mobile & Desktop -->
```

Comment:
- [ ] Kindly make sure your `Home Page`  contains a top section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, a smaller line of text as required for the project, and all the text have the identical `fonts(size && weight)` as on the Behance template.

| Template Design | Your Design |
| ----------------------- | ------------------- |
| ![Screenshot from 2022-09-14 19-16-10](https://user-images.githubusercontent.com/83042742/190808417-4658d1b9-9d02-41b7-b3b3-28b95b6d8d9f.png) | ![Screenshot from 2022-09-16 23-59-51](https://user-images.githubusercontent.com/83042742/190808611-a5fb3979-0fd3-4f94-a234-6d1605a04a3e.png) |